panel/io/server.py

```diff
@@ -601,8 +611,8 @@ class ComponentResourceHandler(StaticFileHandler):
         '_css', '_js', 'base_css', 'css', '_stylesheets', 'modifiers', '_bundle_path'
     ]
 
-    def initialize(self, path: Optional[str] = None, default_filename: Optional[str] = None):
-        self.root = path
+    def initialize(self, path: str | None = None, default_filename: str | None = None):
+        self.root = path or 'root'
```

Comment:
```suggestion
    def initialize(self, path: str | Literal["root"] = "root", default_filename: str | None = None):
        self.root = path
```
This will hint at what happens.